apps/site/lib/site_web/templates/page/_homepage-events.html.eex

```diff
@@ -0,0 +1,19 @@
+<div class="container page-section">
+  <div class="c-standard-fares__heading">
```

Comment:
Now that we're reusing this class, we should probably give it a less specific name

Reply:
I think I'll defer this since there'll be some other sections similar to this that will need to be streamlined...